app/src/main/java/edu/wisc/ece/pinpoint/pages/pins/PinViewFragment.java

```diff
@@ -93,7 +93,7 @@ public void setPinData(Pin pin) {
         } else {
             caption.setText(pin.getCaption());
         }
-        foundCount.setText("12 finds");
+        foundCount.setText(String.format("finds: %d", pin.getFinds()));
```

Comment:
Can we change the format of this to be `%d finds`? I feel like the colon doesn't look as clean and sounds less natural.

Reply:
ye good call